tests/e2e/containers/containers.go

```diff
@@ -154,6 +161,35 @@ func (m *Manager) ExecCmd(t *testing.T, containerName string, command []string,
 			}
 
 			errBufString := errBuf.String()
+			if (errBufString != "" || outBuf.String() != "") && containerName != hermesContainerName {
```

Comment:
Can we add a code comment for whats going on here. I have no idea what these next 20 lines are.

Alternatively extract them into another function?

Reply:
Added code comment here https://github.com/osmosis-labs/osmosis/pull/5598/commits/4a6d477dffb9b1ba38f4b670e12f3f16d9cfe6b8

It basically takes the expected sequence number from the error message and reformats the txn with the expected sequence number. This allows for a single validator to get in multiple txns in the same block, without having to track the sequence number across all accounts.

A little hacky, but worked!